### PR TITLE
Jamesba drop27

### PR DESCRIPTION
**Pull request by jamesba**
Job: https://www.pivotaltracker.com/story/show/169125043

### CODE REVIEW

**Comment by jamesba** [comment]
Should pass CI once https://github.com/bbc/rd-apmm-python-lib-mediatimestamp/pull/41 is merged

File: Jenkinsfile [hunk 1]
```diff
@@ -53,7 +53,6 @@ pipeline {
                     env.lint_result = "FAILURE"
                 }
                 bbcGithubNotify(context: "lint/flake8", status: "PENDING")
-                // Run the linter, excluding build directories (this can also go in the .flake8 config file)
```
Comment (philipnbbc) [inline Jenkinsfile]: Could also remove a comment at line 6 about python 2.7 and a comment at line 90 about using /tmp working directory.

File: Jenkinsfile [hunk 2]
```diff
@@ -183,6 +176,31 @@ pipeline {
...
+                                bbcShouldUploadArtifacts(branches: ["dev"])
+                            }
+                        }
+                    }
+                    steps {
+                        script {
+                            env.artifactoryUpload_result = "FAILURE"
+                        }
+                        bbcGithubNotify(context: "artifactory/upload", status: "PENDING")
+                        bbcTwineUpload(toxenv: "py36")
```
Comment (philipnbbc) [inline Jenkinsfile]: Does `bbcMakeGlobalWheel` need to be called when on the `dev` branch? Maybe this should be done in a separate stage?

It appears that `bbcMakeGlobalWheel` will only be called in the previous stage when on the `master` branch or when `FORCE_PYUPLOAD` is set.

File: mediajson/mutable.py [hunk 3]
```diff
@@ -21,9 +21,6 @@
 NMOSJSONEncoder and NMOSJSONDecoder as your encoder and decoder classes.
```
Comment (philipnbbc) [inline mediajson/mutable.py]: Why not remove `mutable.py`? It doesn't seem worth the maintenance effort to keep it as an option.

File: tox.ini [hunk 4]
```diff
@@ -4,11 +4,12 @@
...
 
 [tox]
-envlist = py27, py36
+envlist = py36
 toxworkdir = {env:TOX_WORK_DIR:{toxinidir}}/mediajson/.tox
 
 [testenv]
 commands = python -m unittest discover -s tests
 deps =
+    -e /home/jamesba/src/rd-apmm-python-lib-mediatimestamp
```
Comment (philipnbbc) [inline tox.ini]: You forgot to remove this.

**Comment by jamesba** [comment]
@philipnbbc removed mutable, and took the opportunity to refactor the code organisation.

**Pull request merged**